sample/confidential_client_certificate_sample.py

```diff
@@ -63,6 +63,7 @@ def acquire_and_use_token():
     result = global_app.acquire_token_for_client(scopes=config["scope"])
 
     if "access_token" in result:
+        print("Token was obtain from:", result["token_source"])  # Since MSAL 1.25
```

Comment:
FYI, @bgavrilMS , this is what I meant by ["the string literals become the real API"](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/610#discussion_r1373509560).

BTW, all the built-in samples in this repo have been updated with this line. So, this will also satisfy the last requirement of #602 .

Reply:
Small typo "obtained"